packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItem.base.tsx

```diff
@@ -26,7 +26,7 @@ const renderCheckMarkIcon = ({ onCheckmarkClick, item, classNames }: IContextual
     // Ensures that the item is passed as the first argument to the checkmark click callback.
     const onClick = (e: React.MouseEvent<HTMLElement>) => onCheckmarkClick(item, e);
 
-    return <Icon iconName={isItemChecked ? 'CheckMark' : ''} className={classNames.checkmarkIcon} onClick={onClick} />;
+    return <Icon iconName={!!item.canCheck && isItemChecked ? 'CheckMark' : ''} className={classNames.checkmarkIcon} onClick={onClick} />;
```

Comment:
To preserve current implementation, I think that it should only if item.canCheck === false. Currently if it's undefined for one item, then it will be visible. That way we don't break people who are relying on check being visible in the current case where canCheck only needs to be true for one item.